src/lib/markbind/src/parsers/componentParser.js

```diff
@@ -196,10 +254,10 @@ function _parseDropdownAttributes(element) {
   const slotChildren = el.children && el.children.filter(child => _.has(child.attribs, 'slot'));
   const hasHeaderSlot = slotChildren && slotChildren.some(child => child.attribs.slot === 'header');
 
+  _warnDeprecatedAttribute(element, 'text', 'header');
   // If header slot is present, the header attribute has no effect, and we can simply remove it.
   if (hasHeaderSlot) {
     delete el.attribs.header;
```

Comment:
Should we warn here too? 

If user has header slot and header/text attr

Reply:
Wasn't part of the original issue but good to include in the fix. 👍

Reply:
Hmm I think having a separate function for this kind of warnings (`ignoredAttribute` maybe?) might be useful

Reply:
I still feel it's overkill, the case for this is too specific to abstract a function out. i.e. that if there is a X slot, therefore attribute Y has no effect.